opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -28,13 +28,17 @@ message Metric {
   // The descriptor of the Metric.
   MetricDescriptor metric_descriptor = 1;
 
-  // One or more timeseries for a single metric, where each timeseries has
-  // one or more points.
-  repeated TimeSeries timeseries = 2;
-
   // The resource for the metric. If unset, it may be set to a default value
   // provided for a sequence of messages in an RPC stream.
-  opentelemetry.proto.resource.v1.Resource resource = 3;
+  opentelemetry.proto.resource.v1.Resource resource = 2;
+
+  // data is a list of one or more TimeSeries for a single metric, where each timeseries has
+  // one or more points. Only one of the following fields is used for the data, depending on
+  // the type of the metric defined by MetricDescriptor.type field.
+  repeated Int64TimeSeries int64_timeseries = 3;
+  repeated DoubleTimeSeries double_timeseries = 4;
+  repeated HistogramTimeSeries histogram_timeseries = 5;
```

Comment:
should we call it doubleHistogram? In case we will decide to have int64 histogram in future... sorry if it was already discussed before

Reply:
this comments is not super relevant for this change =)

Reply:
That's a good question. We can add int64 histogram later, but you are right that it would be nice to get the name for this histogram right, so that in the future we don't end up with assymetric naming like `Histogram` and `Int64Histogram`. If there is a feeling int64 histogram may be needed in the future, it may be worth renaming Histogram to `DoubleHistogram` now (or perhaps `HistogramOfDoubles` as a better name? Neither name is very nice to me).

Reply:
let's evaluate that later.

Reply:
https://github.com/open-telemetry/opentelemetry-proto/issues/35

Reply:
@SergeyKanzhelev thanks for creating the issue. Can we merge this PR and if we decide to rename the message we can do in a follow-up PR?

Reply:
I was waiting for a day for more comments... Ill merge by EOD ok?

Reply:
Sounds good, thanks.